packages/recomponents/src/components/r-date-input/r-date-input.vue

```diff
@@ -193,8 +193,10 @@
                         start: moment(date.start).utc().tz(this.timezone, true),
                         end: moment(date.end).utc().tz(this.timezone, true),
                     };
+                } else if (this.type === DateInputType.date) {
```

Comment:
Will this change bring issues to Recomm? that is probably expecting the UTC time? I wonder if would it be possible to have this `if` to include something like `this.type === DateInputType.date && !this.timezone)` the only issue with this is that if we don't include the timezone the component returns `undefined`.

Reply:
I'm preparing the PR for it. We don't use type `date` too much.

Reply:
IMO the `date` type is not supposed to return the date with time value. It should return absolute only date value.